triframe/logging.py

```diff
@@ -14,6 +14,9 @@ def log_actor_choice(option: Option) -> LogWithAttributesRequest:
     message = ""
     style = log_styles["actor_no_function"]  # Default style for no function call
 
+    if option.thinking_blocks:
+        for thinking_block in option.thinking_blocks:
+            message += f"Thinking:\n{thinking_block['thinking']}\n"
```

Comment:
`thinking_blocks` includes `redacted_thinking`, so this would throw a `KeyError`.

I notice that this wasn't caught by type checks. Probably because the `Option.thinking_blocks` type is too loose. I'll make a suggestion on the `Option` class directly.